src/calibration/reprapfirmware_lsq.py

```diff
@@ -0,0 +1,555 @@
+import math
+import logging
+
+__author__ = 'Pedro'
+"""
+Derived from https://github.com/payala/DeltaTuner by @payala
+
+Delta printer least-squares calibration calculator
+
+This calculator implements the least-squares delta calibration algorithm that RepRapFirmware has built-in.
+I have provided it as a service for those not running RepRapFirmware.
+You can choose to calibrate the following parameters:
```

Comment:
can be fixed

Reply:
This is original author docstring. What if we just remove the last sentence?